packages/data-store/src/types/contact.ts

```diff
@@ -11,9 +11,9 @@ export enum ConnectionRoleEnum {
 }
 
 export enum ConnectionTypeEnum {
-  OPENID = 'openid',
-  DIDAUTH = 'didauth',
-  SIOPV2_OIDC4VP = 'siopv2+oidc4vp',
+  OPENID_CONNECT = 'OpenIdConnect',
```

Comment:
Suggest to change the value (not the key) to OIDC as that is the registered/normal abbreviation. If we still need to mention OpenID Connect in place, make sure to get the casing correct, as these are registered/well-known names.

Reply:
changed